search-app/src/cmr/search/services/parameters/converters/spatial.clj

```diff
@@ -1,6 +1,7 @@
 (ns cmr.search.services.parameters.converters.spatial
   "Contains parameter converters for spatial parameters"
-  (:require [cmr.common-app.services.search.params :as p]
+  (:require [cmr.common.log :refer [debug error info warn]]
```

Comment:
We have been converting namespace declarations as we see them to fit the style of this article. Join us!

https://stuartsierra.com/2016/clojure-how-to-ns.html